site/docs/v1/tech/apis/_application-request-body.adoc

```diff
@@ -19,6 +19,11 @@ True if CleanSpeak username moderation is enabled.
 [field]#application.data# [type]#[Object]# [optional]#Optional#::
 An object that can hold any information about the Application that should be persisted.
 
+[field]#application.formConfiguration.adminRegistrationFormId# [type]#[UUID]# [default]#defaults to the default Admin Registration Form Id# [optional]#Optional# [since]#Available since 1.20.0#::
```

Comment:
This will default to the formId in use by the FusionAuth application. So the default is not a one value answer, so we generally move the default down below the field description to better describe it in full. 

See https://fusionauth.io/docs/v1/tech/apis/audit-logs/ `dateTimeSecondsFormat` for example.